### DISCUSSION TITLE
Wemos Lolin32 SD_MMC PN5180 PCB

### DISCUSSION THREAD

**Discussion by tuniii** [biologist79/ESPuino#96] [General]
Hi,
wäre es möglich das KiCad-Projekt für die Platine "Wemos Lolin32 SD_MMC PN5180" in das Repo zu pushen?
Ich würde gerne die Platine auf ein ESP32-**S2** Dev Board anpassen. Ich denke, dass der µC eventuell eine gute Alternative für dieses Projekt wäre. Ein Aufbau am Steckbrett sieht schon sehr vielversprechend aus.

**Comment by Soehm** [comment]
Hey,

Ich habe keine direkt Antwort auf deine Frage: bist du im Forum aktiv? Dort gab es mal einen kurzen Beitrag zum ESP-S2. Dieser scheint nur bedingt geeignet zu sein (Single-Core, kein Bluetooth usw).

Beste Grüße
Stefan

**Reply by tuniii** [reply]
Hi Stefan,

leider kann ich mich nicht über Github im Forum anmelden, sonst hätte ich dort die Frage gestellt.

Grundsätzlich sollte der S2 geeignet sein. Das mit dem "fehlenden" Bluetooth ist für mich kein Problem, da dies aktuell kein Use Case für mich ist und auch im Projekt aktuell als Optional eingestuft ist.

Was mich am S2 reizt ist das bessere Power Management beim Wifi und die Anzahl der GPIOs. Für den ESP32 habe ich für ein anderes Projekt den Arduino Core angepasst, um mehr Strom zu sparen. Allerdings gibt es dabei ein paar Sachen zu beachten. Aufgrund der verwendeten Peripherie wäre das hier keine Option. Beim S2 ist das Wifi Power Management in der Hardware umgesetzt und somit direkt "nutzbar".

Ein single Core sehe ich nicht als Problem. Aktuell läuft der Aufbau mit dem S2 stabil und es gibt meistens noch Optimierungspotenzial. Ich werde auch Laufzeitmessungen machen, um zu schauen, wie langweilig es der CPU ist.

Zur Nutzung des S2 musste ich bisher nur wenige Änderungen machen. Bei den verwendeten Libs habe ich bisher nur die ESP32-audioI2S ein wenig anpasst.

Ich denke man kann dem S2 ein Chance geben. Der offizielle Arduino Core wird kommen und bis dahin teile ich gerne meine Erfahrung.

Viele Grüße
Martin

**Reply by biologist79** [reply]
Moin,

was passiert denn, wenn du dich beim Forum mit deinem Github-Account anmelden willst? Ich hatte das eingerichtet, getestet und das lief eigentlich. Zur Not: Man kann sich dort auch "einfach so" anmelden :-)

Zu deiner Frage: Ich lade es später mal woanders hin hoch (temporär). Ich habe halt wegen Namensstreitigkeiten alles umbenannt, aber KiCad hat irgendwelche internen Verweise, die ich nicht raffe :-) Jedenfalls reicht es nicht, einfach nur die Dateien umzubenennen. Da habe ich es vor ner Weile aus dem Repository gekickt.

Aber interessant zu hören, dass der S2 auch läuft.

**Reply by tuniii** [reply]
Folgende Fehlermeldung kommt beim Registrieren über Github:
![image](https://user-images.githubusercontent.com/23635855/111062592-19f93000-84aa-11eb-995a-3a815ce8083c.png)
Das Anmelden bei anderen Seiten über Github hat bisher immer funktioniert.

Vielen Dank schon mal für das KiCad Projekt :-)

**Reply by biologist79** [reply]
https://espuino.de/temp_files/Lolin32_MMC_PN5180.zip

**Reply by biologist79** [reply]
Teste bitte mal den Github-Login im Forum. Das müsste jetzt wieder klappen.

**Reply by tuniii** [reply]
Hat funktioniert 👍 .
